packages/bubble-dev/start-preset/src/plugins/sync-state.ts

```diff
@@ -6,7 +6,7 @@ const PORT = 3001
 export default plugin<{}, void>('syncState', ({ logMessage }) => async () => {
   const { default: WebSocket } = await import('ws')
 
-  await new Promise((resolve, reject) => {
+  await new Promise<void>((resolve, reject) => {
```

Comment:
To be honest I don't understand where the error came from and why this is fixing it. Do you know or you tried several things and this solved it (which is fine)? 😅

Reply:
Nevermind! I read the PR description! Approved! ✅

Reply:
Although, on complete side note, I'm still puzzled why minor version of typescript contains breaking changes? 🤷‍♀️

Reply:
I guess they consider the bump to *.1.* to be some serious bump 😊